dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/FieldResource.java

```diff
@@ -295,24 +330,57 @@ public Response updateContentTypeFieldByVar(@PathParam("typeId") final String ty
 
 
 	@DELETE
+	@JSONP
+	@NoCache
+	@Produces({ MediaType.APPLICATION_JSON, "application/javascript" })
+	public Response deleteFields(@PathParam("typeId") final String typeId, final String[] fieldsID, @Context final HttpServletRequest req)
+			throws DotDataException, DotSecurityException {
+
+		final InitDataObject initData = this.webResource.init(null, false, req, false, null);
+		final User user = initData.getUser();
+
+		Response response = null;
+		try {
+			final List<String> deletedIds = new ArrayList<>();
+
+			for (final String fieldId : fieldsID) {
+				try {
+					Field field = fieldAPI.find(fieldId);
+					fieldAPI.delete(field, user);
+					deletedIds.add(fieldId);
+				} catch (NotFoundInDbException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)